rust/cubesql/cubesql/src/compile/engine/df/scan.rs

```diff
@@ -475,31 +478,28 @@ impl ValueObject for JsonValueObject {
         Ok(self.rows.len())
     }
 
-    fn get<'a>(
-        &'a mut self,
+    fn get(
+        &mut self,
         index: usize,
         field_name: &str,
     ) -> std::result::Result<FieldValue, CubeError> {
-        let option = self.rows[index].as_object_mut();
-        let as_object = if let Some(as_object) = option {
+        let as_object = if let Some(as_object) = self.rows[index].as_object() {
             as_object
         } else {
             return Err(CubeError::user(format!(
                 "Unexpected response from Cube, row is not an object: {:?}",
                 self.rows[index]
             )));
         };
-        let value = as_object
-            .get(field_name)
-            .unwrap_or(&Value::Null)
-            // TODO expose strings as references to avoid clonning
-            .clone();
+
+        let value = as_object.get(field_name).unwrap_or(&Value::Null);
+
         Ok(match value {
-            Value::String(s) => FieldValue::String(s),
+            Value::String(s) => FieldValue::StringRef(s),
             Value::Number(n) => FieldValue::Number(n.as_f64().ok_or(
```

Comment:
There is an additional boost here (we don't clone it), while n is used as a reference. 

<img width="537" alt="image" src="https://github.com/user-attachments/assets/a970e118-f195-4960-a122-5dd3e888f7ef">